IFC4x3/Sections/Core data schemas/Schemas/IfcProductExtension/Entities/IfcFacilityPartCommon/DocEntity.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocEntity xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="IfcFacilityPartCommon" UniqueId="7cdf39c9-eebf-4575-a208-dfab7b52147c" DiagramNumber="2" BaseDefinition="IfcFacilityPart" EntityFlags="32">
+	<DiagramRectangle xsi:type="DocRectangle" X="1909,3333740234375" Y="533,3333129882812" Width="400" Height="100" />
+	<Attributes>
+		<DocAttribute Name="PredefinedType" UniqueId="e01c890f-0104-4d79-9760-3a9519663b57" DefinedType="IfcFacilityPartCommonTypeEnum" AttributeFlags="1">
+			<Documentation>A list of types to further identify the object. Some property sets may be specifically applicable to one of these types.
+
+&gt;NOTE If the object has an associated IfcTypeObject with a PredefinedType, then this attribute shall not be used.</Documentation>
+			<Definition>
+				<DocEnumeration xsi:nil="true" href="IfcFacilityPartCommonTypeEnum" />
+			</Definition>
+		</DocAttribute>
+	</Attributes>
+	<WhereRules>
```

Comment:
Same comment as for all objects lacking a corresponding type object.

Reply:
Have pushed changes for spatial elements.